config/crd/bases/apps.foundationdb.org_foundationdbbackups.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.6.1
+    controller-gen.kubebuilder.io/version: v0.6.0-beta.0
```

Comment:
Can you fix that please? You have to delete `controller-gen` and then run it again.

Reply:
done